lib/sinon/spy.js

```diff
@@ -38,7 +38,7 @@ function createSpy(func, arity) {
         name = functionName(funk);
     }
 
-    var proxy = createProxy(funk, arity);
+    var proxy = createProxy(funk, funk);
```

Comment:
This reads like a mistake ... is it though?

Reply:
No, it's on purpose. `createProxy` now takes two function arguments, the fake and the original function. In case of a spy there is no difference between the two.

Reply:
👍